architecture/additional_concepts/authorization.adoc

```diff
@@ -228,17 +228,412 @@ TIP: Remember that xref:authentication.adoc#users-and-groups[users
 and groups] can be associated with, or _bound_ to, multiple roles at the same
 time.
 
-Cluster administrators can visualize these roles, including a matrix of the
-verbs and resources each are associated using the CLI to
+Cluster administrators can investigate policy roles by using the `oc describe` command, which displays a matrix of the `Verbs`, `Non-Resource URLs`, `Extension`, `Resource Names`,`API Groups`, and `Resources` that apply to the policy role.
+
+----
+$ oc describe clusterrole <role>
+----
+
+The rule verbs apply to the objects listed in that row. For example, if the `create` verb is applied to the `serviceaccounts` resource, the holder is allowed to create service accounts but no other resource unless otherwise specified in the role matrix. +
+The verbs are described in the following table.
```

Comment:
There seems to be a rogue + before "The verbs are..."

Reply:
That should render as a new line without whitespace - unless asciibinder renders differently to ccutil?